high_card.rb

```diff
@@ -3,3 +3,39 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+
+ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+suits = [ "hearts", "spades", "clubs", "diamonds" ]
+
+deck=[]
+
+ranks.each do |rank|
+  suits.each do |suit|
+    deck.push([rank,suit])
+  end
+end
+
+deck.shuffle!
+
+players = []
+
++puts players.length.to_s = "{n} players so far. Enter a player name, or type 'play':"
++name = gets.chomp.to_s
++if name === "play"
++  players.push(name)
++end
+
+cards = players.map do |card|
+  card|:score|
+end
```

Comment:
I don't think pipes is what you want on line 29. We want to be able to map the values of the card, might be something more like `card[0]`. But that's only if we have access to a card array when we map. We need some way to get a card to a player. then map that cards value to each player and compare.